src/remoteCollection/handleCollectResponse.js

```diff
@@ -140,7 +140,12 @@ function generateSamples(collectRes) {
    */
   const func = RefocusCollectorEval.getTransformFunction(tr, status);
   if (func) {
-    return RefocusCollectorEval.safeTransform(func, args);
+    try {
+      return RefocusCollectorEval.safeTransform(func, args);
+    } catch (err) {
+      const errorMessage = `Transform error: ${err.message} (${collectRes.preparedUrl})`;
```

Comment:
Need to be careful with this, because it gets set as messageBody but depending on the values of err.message and preparedUrl the length may exceed the max size of messageBody--should probably add some truncation logic somewhere along the way within the collector so that Refocus doesn't just reject the sample.